department-of-veterans-affairs/abd-vro
# MCP-2299 fetchPdf Error Code Change

## What was the problem?

Fetching an invalid submission ID returns a 404 when it should be a 400

Associated tickets or Slack threads:

- [MCP-2299](https://amida.atlassian.net/browse/MCP-2299)

## How does this fix it?[^1]

Update to the proper code

## How to test this PR
- Use swagger to fetch a pdf by invalid claim submission id. The response code should be a 400
